ui/pages/unlock-page/unlock-page.container.js

```diff
@@ -89,12 +94,25 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     ...restOwnProps,
     onRestore: onImport,
     onSubmit: ownPropsSubmit || onSubmit,
-    history,
+    navigate,
     location,
+    navState,
   };
 };
 
-export default compose(
-  withRouter,
+const UnlockPageConnected = compose(
+  withRouterHooks,
   connect(mapStateToProps, mapDispatchToProps, mergeProps),
 )(UnlockPage);
+
+// Inject navState from NavigationStateContext for v5-compat navigation
+// eslint-disable-next-line react/prop-types
```

Comment:
I cannot reproduce this error on my side 🤔 
Can you check if this eslint-disable is necessary?

If you still get the error, it maybe can be possible to avoid having it by defining something generic like this:
```typescript
import PropTypes from 'prop-types';

// ... ... ...

UnlockPageWithNavState.propTypes = {
  ...PropTypes.object
};
```